rs/nns/governance/src/lib.rs

```diff
@@ -940,6 +942,27 @@ pub fn encode_metrics(
         if let Some(public_neuron_subset_metrics) = public_neuron_subset_metrics {
             public_neuron_subset_metrics.encode("public", "have visibility set to public", w)?;
         }
+
+        if let Some(still_losing_voting_power_neuron_subset_metrics) =
+            still_losing_voting_power_neuron_subset_metrics
+        {
+            still_losing_voting_power_neuron_subset_metrics.encode(
+                "still_losing_voting_power",
+                "have deciding voting power < potential voting power (but still positive) due \
+                 to not refreshing recently enough.",
+                w,
+            )?;
+        }
+
+        if let Some(done_losing_voting_power_neuron_subset_metrics) =
+            done_losing_voting_power_neuron_subset_metrics
+        {
+            done_losing_voting_power_neuron_subset_metrics.encode(
+                "done_losing_voting_power",
+                "have 0 deciding voting power due to not refreshing recently enough.",
+                w,
+            )?;
+        }
```

Comment:
This is where the "rubber meets the road", so to speak. This is where the data gets written out in Prometheus format. Not terribly interesting. Looks just like the code directly above this.